components/style/properties/declaration_block.rs

```diff
@@ -935,13 +935,23 @@ impl<'a, 'b, 'i> AtRuleParser<'i> for PropertyDeclarationParser<'a, 'b> {
     type Error = SelectorParseError<'i, StyleParseError<'i>>;
 }
 
+fn is_non_mozilla_vendor_identifier(name: &str) -> bool {
+    (name.starts_with("-") && !name.starts_with("-moz")) ||
```

Comment:
I based this on https://dxr.mozilla.org/mozilla-central/rev/7d92f47379da55adf4da6d0b16398f5c629cf949/layout/style/nsCSSParser.cpp#1669-1676 which uses a case-sensitive check.